src/message/html/HtmlTagPre.js

```diff
@@ -3,8 +3,12 @@ import { View } from 'react-native';
 
 import HtmlTagSpan from './HtmlTagSpan';
 
-export default ({ style, data, childrenNodes, cascadingStyle }) => (
+export default ({ style, data, childrenNodes, cascadingStyle, doNarrow }) => (
```

Comment:
Maybe we want to turn that into 
```js
{ style, ...restProps }
<HtmlTagSpan {...restProps} />
```
and possiblt through out the other components. 
Seems more manageable the more handlers we pass.